pravega/pravega
# Issue 4516: Enforce correct copyright statement through Checkstyle

Signed-off-by: Flavio Junqueira (fpj) <fpj@apache.org>

**Change log description**  
Changes the checkstyle configuration to validate the copyright statement in license headers.

**Purpose of the change**  
Fixes #4516 

**What the code does**  
There is no code change only a change to the checkstyle configuration to validate the copyright statement:

```
  <module name="RegexpHeader">
    <property name="header" value="/\*\*\n * Copyright \(c\) Dell Inc\.\, or its subsidiaries\. All Rights Reserved\."/>
  </module>
```

**How to verify it**  
It should build correctly once #4515 is merged. Before #4515 is merged, the build will fail.
